bundle/edu.gemini.itc.shared/build.sbt

```diff
@@ -26,7 +26,6 @@ OsgiKeys.bundleActivator := Some("edu.gemini.itc.shared.osgi.Activator")
 OsgiKeys.bundleSymbolicName := name.value
 
 OsgiKeys.exportPackage := Seq(
-  "edu.gemini.itc.service",
```

Comment:
This package doesn't exist in this bundle, but it does in `edu.gemini.itc` and I tried to export it from there and it didn't work because it was already exported from here but was empty. This took a really long time to figure out.